routes/users.js

```diff
@@ -25,34 +44,96 @@ router.route('/add').post((req, res) => {
 		.catch((err) => res.status(400).json(err));
 });
 
-router.route('/').delete((req, res) => {
-	User.findOneAndDelete({ firebase_id: req.body.firebase_id })
-		.then(() => res.json('User ' + req.body.firebase_id + ' was deleted!'))
-		.catch((err) => res.status(400).json(err));
+/**
+ * --------------------- PUT (update existing objects) ----------------------
+ */
+
+/**
+ * Sets user given by firebase id as an organizer
+ */
+router.route('/makeOrganizer').put(async (req, res) => {
+	const { firebase_id } = req.body;
+
+	if (!firebase_id)
+		return res
+			.status(400)
+			.json('Required firebase_id not provided in request body.');
+
+	try {
+		let user = await User.findOne({ firebase_id: firebase_id });
+
+		if (!user)
+			return res
+				.status(404)
+				.json('Could not find user specified by firebase_id.');
+
+		user.isOrganizer = true;
```

Comment:
No need to have two, let's just do one endpoint and pass in true/false so we don't have too many endpoints